src/terminal/adapter/ut_adapter/inputTest.cpp

```diff
@@ -482,8 +482,16 @@ void InputTest::TerminalInputModifierKeyTests()
             // Alt+Key generates [0x1b, Ctrl+key] into the stream
             // Pressing the control key causes all bits but the 5 least
             // significant ones to be zeroed out (when using ASCII).
-            if (AltPressed(uiKeystate) && ControlPressed(uiKeystate) && ch >= 0x40 && ch < 0x7F)
+            if (AltPressed(uiKeystate) && ControlPressed(uiKeystate))
             {
+                if (ch == 0)
+                {
+                    // There are a few vkeys that MapVirtualKeyW() above turns into 0,
+                    // while TerminalInput is still capable of processing them.
+                    // The important ones are tested in DifferentModifiersTest/TerminalInputNullKeyTests.
+                    continue;
+                }

```

Comment:
@zadjii-msft In order for the tests to work again I disabled tests for certain Ctrl+Alt combinations as you can see here.
We could make the tests work if we pull in the `ToUnicodeEx()` logic into this file as well. If we do that though we should probably really create a helper file that contains this function separately. 😄

On the other hand I personally kinda doubt that this if condition makes this unit test any worse, because to fix this unit test we have to basically add a identical copy of the `TerminalInput::_CharacterFromAltCtrlKeyEvent` method.
And IMHO that way we'd just test for the current state of affairs of which OS methods we use instead for the actual results that we'd like to see for each character, you know? 🤔

Reply:
Yea this is fine with me